assets/scss/hero.scss

```diff
@@ -1,3 +1,27 @@
+.item-hero .user-buttons .btn {
```

Comment:
I had to move this selector since stylelint was complaining about descending selectors
i.e. .item-hero .user-buttons .btn should come before .jumbotron.browse-jumbotron .container .card .btn